### DISCUSSION TITLE
I can't use Photopea anymore because 'Change consent' button is not there

### DISCUSSION THREAD

**Discussion by Sputnik2023** [photopea/photopea#6051] [General]
I've been using Photopea forever now. And I've always been super happy with it. 

But now suddenly, I get this popup with 'Please let us to show ads (change consent on the right), or get a premium account". There is no 'change consent' button anywhere on the page. I tried to get a premium account, but when I click the Account button, I get the same pop-up, and I can't create a premium account. I've tried all kinds of solutions that I found on Reddit, but nothing works.  Not on Safari and not in Google Chrome. Can you please please help with this?  I really need Photopea. 

I use MacOs, on Safari and I've also tried Chrome.

**Comment by photopea** [comment]
our ad provider has a bug. It tells Photopea that the user refuses to watch ads, when the user is from Canada, without even asking the user.

I decided to allow everyone who refuses to see ads to still use Photopea, until this issue is fixed. So you should be able to use Photopea now.
